otcextensions/sdk/dns/v2/floating_ip.py

```diff
@@ -9,32 +9,99 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-from openstack import resource
+import six
```

Comment:
six was only required for keeping py2 compatibility. Now we do not need it and instead we actually should get rid of it through the code. Please replace this usage with urllib library directly